### DISCUSSION TITLE
Is it possible to remove the "on", "via", "took" words from the prompt?

### DISCUSSION THREAD

**Discussion by jcayzac** [starship/starship#1454] [Q&A] [answered]
Those take space for nothing.

**Comment by andytom** [comment, marked as answer]
Hi @jcayzac, this is possible for some modules but not all, as part of the [v0.45.0](https://github.com/starship/starship/milestone/3) release we are changing how the prefixes work to make it easier to configure them (among other things).

**Reply by jcayzac** [reply]
Thanks @andytom !
I <3 how I can use the same prompt in Bash and Fish

**Comment by Bibelo** [comment]
I don't understand this keyword "on"

user@hostname
user on hostname

4 chars instead of 1

Why would you do that?
